.github/workflows/docs.yaml

```diff
@@ -0,0 +1,44 @@
+name: CI Docs
+
+on:
+  push: # Run on pushes to the default branch
+    branches: [main, docs_job]
+  pull_request_target: # Also run on pull requests originated from forks
+    branches: [main, docs_job]
+  release:
+    types: ['published']
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+
+  Deploy-Pages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.ref }}
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          architecture: "x64"
+
+      - name: Install packages and dependencies
+        run: pip install hatch
+
+      - name: Deploy Docs
+        run: |
+          git config user.name "Pankaj Singh"
+          git config user.email "pankaj.singh@astronomer.io"
```

Comment:
@tatiana / @pankajkoti Do we have any bot for this that we can use

This look like may need: https://github.com/jimporter/mike?tab=readme-ov-file#deploying-via-ci

Reply:
I believe it is very important we use a bot for this - if we don't have one yet, we can create one. Please, confirm with @phanikumv , @pankajastro - we should make this change before merging.

Reply:
yes, we shouldnt use personal details. @pankajastro could you request IT to create a `oss-integrations-bot` user. Then we can add it to the repo and test it

Reply:
Look like it also work with dummy user :)